### ISSUE TITLE
[Documentation / Wiki] Should we use https://github.com/ruby-gnome/ruby-gnome/wiki ?

### ISSUE THREAD

**Issue by rubyFeedback** [ruby-gnome/ruby-gnome#1632]
Hey there kou,

I remember that the old ruby-gtk / ruby-gnome used to have a wiki. I think it was originally
a hiki-wiki; I vaguely remember that I was able to edit things. If I recall correctly you gave me
the passwords many years ago, too. :)

I noted down these old URLs, such as http://ruby-gnome2.sourceforge.jp/hiki.cgi? or
https://ruby-gnome2.osdn.jp/hiki.cgi?Gtk%3A%3AEntry - the latter one for Gtk::Entry, for
instance.

Alright, so it is gone, no need to be sad about it. What if we would create a new wiki?

We can use github wiki pages such as here:

https://github.com/ruby-gnome/ruby-gnome/wiki

Ideally we can have several people be able to contribute to the wiki; I think having 
access to the wiki for those few folks may initially suffice. Over time, e. g. some months,
we can slowly begin to document things in ruby-gtk; perhaps for both ruby-gtk3 and
ruby-gtk4.

We can also decide lateron on any format of that, once we have some content to show
there. My recommendation would be to focus on these parts on the wiki here:

- API documentation e. g. how to use Gtk::Entry.new and so forth.
- Working examples, ideally standalone. These could also be supplied by the ruby-gtk
gem as such, but it is also ok to showcase smaller snippets on the wiki, to help illustrate
how something works in ruby-gtk.
- A tutorial. This one could be a guided walk-through, or it could also be more integrated
into the API itself, or a combination of both, e. g. first document the API of Gtk::Entry,
then showcase a few examples, and then also give some higher-level recommendation
how to best use it, how to combine it with different things and so forth.

Everything has to start at some point; I can begin to write a little bit here and there, and
perhaps more folks can join once they see that things progress. But I think for this to
happen, it would be best to ask for your ok on it, simply because you are the boss guy
in regards to the C-interface (similar to kojix2 and the ruby-libui-specific code; I know 
a bit of ruby, but my C knowledge is still very limited. If only learning C would be as much
fun as writing ruby code ...).

Anyway, TL;DR: would you a) be ok with a wiki in regards to ruby-gtk, and b) should we then
use the github wiki for it?

(Also, do you happen to know if there is anyone who may have an off-line copy of the old
documentation? I collected a lot in my own local knowledge base, so I can go through 
that, which also helps when I add to the wiki, but I don't have everything, so any off-line
docu may still be useful. Some parts could be re-used, others are probably now long 
dead and no longer useful, but a few old tutorials may still contain a bit of useful
information.)

**Comment by Beagle123** [comment]
I would second this.

I can't find any documentation on ruby/gtk.  I would love to get that old wiki back.  I know that there was a tarball file with all the content in it.  Can we get the last tarball?  I agree that we should put up the old docs.  They're better than nothing.  It would be great to have anything.    

Also, kou, I'm hoping to push my program to rubygems, but it will crash in windows due to the pixbuf bug that you already fixed.  My program will work as soon as you push the next gtk3 rubygem.  Can you push gtk3?

Thanks for all your great work.  And please let me know if you have a copy of the old tarball for the wiki.

Eric
